support-frontend/cloud-formation/cfn.yaml

```diff
@@ -278,7 +278,8 @@ Resources:
     Properties:
       AlarmActions:
       - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:reader-revenue-dev
-      AlarmName: !Sub No healthy instances for support-frontend in ${Stage}
+      AlarmName: !Sub HIGH severity - No healthy instances for support-frontend in ${Stage}
```

Comment:
What do you think about alarming on <3 rather than 0 healthy instances?  I think there's an argument for an alarm if we are down to 1 healthy - or cycling instances.

Reply:
yeah good idea on membership and subs we have a less than the correct number for 10 minutes, as well as a no healthy instances for 1 minute.  This is because of being burnt in the past not finding out instances couldn't start until it was too late.
I think I will add a card for that.

Reply:
the card is https://trello.com/c/mIRQe53c/2416-add-alert-for-less-than-required-instances-of-support-frontend-for-10-min-period